messaging/src/main/java/org/axonframework/messaging/correlation/MessageOriginProvider.java

```diff
@@ -30,6 +30,7 @@
 public class MessageOriginProvider implements CorrelationDataProvider {
     private static final String DEFAULT_CORRELATION_KEY = "correlationId";
     private static final String DEFAULT_TRACE_KEY = "traceId";
+    private static final String DEFAULT_TRACE_PARENT_KEY = "traceparent";
```

Comment:
How come you've expanded the responsibilities of the `MessageOriginProvider` to carry a `traceparent`, while this field is only present if an actual implementation of tracing is provided?
Wouldn't it be better suited in an additional `CorrelationDataProvider`. Which we do set when a none-noop variant of tracing is selected, for example.

Reply:
This is no longer necessary. I am reverting these changes.